templates/layout/page.html.twig

```diff
@@ -33,7 +35,12 @@
 <section class="main-content ecl-u-mv-l">
   <div class="ecl-container">
     <div class="ecl-row">
-      <div class="ecl-col">
+      {% if page.sidebar %}
+        <div class="ecl-col-12 ecl-col-lg-3">
```

Comment:
According to https://ec.europa.eu/component-library/ec/templates/pages/search/code/ this div should be an `aside` and we should change the main by a `section`.